### PR TITLE
Add ability to use gog level type in server.properties

### LINKED ISSUE #3511: Cant seem to get GOG world on 1.16.3 server
I've used GOG in previous versions with no issues at all but I cant seem to get a GOG world on my 1.16.3 dedicated server. It always comes up as a standard minecraft world, even when only using garden of glass and the other necessary mods needed for gog to work. Using latest versions of necessary mods, I have leveltype=botania-skyblock in server properties and yes have deleted the world since changing the server properties to that. What am I missing? Thanks in advance for your help.

### PR DESCRIPTION
**Pull request by MelanX**
Creating a world on server with level type set to "botania-skyblock" will now create the GoG skyblock world. 
Closes #3511

### CODE REVIEW

**Comment by williewillus** [comment]
Use mixins instead of AT
And if that code is copied of modeled after vanilla code specify where with a // [VanillaCopy]

**Comment by MelanX** [comment]
Done (and it's not copied or modeled after vanilla code)

File: src/main/java/vazkii/botania/common/Botania.java [hunk 1]
```diff
@@ -285,6 +287,9 @@ private void serverAboutToStart(FMLServerAboutToStartEvent event) {
 					+ " \"Do not Override\". Whoever had the brilliant idea of overriding it needs to go"
 					+ " back to elementary school and learn to read. (Actual classname: " + clname + ")");
 		}
+		if (Botania.gardenOfGlassLoaded) {
+			WorldTypeUtil.setupForDedicatedServer((DedicatedServer) event.getServer());
```
Comment (williewillus) [inline src/main/java/vazkii/botania/common/Botania.java:291]: this seems unsafe, what if it's an integrated server?

Comment (MelanX) [reply src/main/java/vazkii/botania/common/Botania.java:291]: true, fixed it :)

**Comment by MelanX** [comment]
I think the forge update 35.1.5 made this unnecessary.

**Comment by Hubry** [comment]
Superceded by d04eb2362bec6f1849660496884f9ef4fd7ced22

**Pull request closed without merging**